tools/bqtag/bqtag/bqtag.py

```diff
@@ -0,0 +1,800 @@
+                                    "Array(SELECT AS VALUE STRUCT("
+                                    + ", ".join(self._rec_generate_query(
+                                        children=child.children,
+                                        parent_name = child.name
+                                    )
+                                )
+                        + ") FROM UNNEST("
+                        + child.name
+                        + ") as "
+                        + child.name
+                        + ")  as "
+                        + child.name)
+                    else:
+                        db.append(
+                                    "Array(SELECT AS VALUE STRUCT("
+                                    + ", ".join(
+                                            self._rec_generate_query(
+                                                children=child.children,
+                                                parent_name = child.name
+                                            )
+                                        )
+                                    + ") FROM UNNEST("
+                                    + parent_name
+                                    + "."
+                                    + child.name
+                                    + ") as "
+                                    + child.name
+                                    + ")  as "
+                                    + child.name
+                                )
+                else:
+                    if parent_name == "":
+                        db.append(
+                                    "STRUCT("
+                                    + ", ".join(
+                                                    self._rec_generate_query(
+                                                        children=child.children,
+                                                        parent_name=child.name
+                                                    )
+                                                )
+                                    +  ")  as "
+                                    + child.name
+                                )
+                    else:
+                        db.append(
+                                    "STRUCT("
+                                    + ", ".join(
+                                                    self._rec_generate_query(
+                                                        children=child.children,
+                                                        parent_name=parent_name
+                                                        + "."
+                                                        + child.name
+                                                    )
+                                                )
+                                    + ")  as "
+                                    + child.name
+                                )
+
+        return db
+
```

Comment:
Two lines separating classes (remove extra blank lines)

Reply:
Changed to:

```
        return db


###########################################################################
# Main class to update tags and create views
###########################################################################
class BQTableView:
```